compiler/z/codegen/OMRCodeGenerator.cpp

```diff
@@ -2335,6 +2335,12 @@ OMR::Z::CodeGenerator::doBinaryEncoding()
       cursor = cursor->getNext();
       }
 
+   if (comp()->getOption(TR_EntryBreakPoints))
+      {
+      TR::Node *node = comp()->getStartTree()->getNode();
```

Comment:
Due to [Extensible Classes](https://github.com/eclipse/omr/blob/master/doc/compiler/extensible_classes/Extensible_Classes.md) in the OMR code generators you'll have to use `self()->` to call any of the functions on the class, here and in the `if` two lines above.

Reply:
Fixed in dc7cc42